### PR TITLE
Migration: migration-guide & whats-new updates

### PR DESCRIPTION
**Pull request by guardrex**
Ready for a look :eyes:

### Notes
Entered in the [Migration Project](https://github.com/dotnet/docs/projects/8) for investigation and usually dealt with by converting to inline code.

#### framework/whats-new/obsolete-members.md
`DragDropHelper.SetDragDropMovedViewElements` depreciated ... find `DoDragMove` replacement.

#### docs/framework/whats-new/obsolete-members.md
The table "Assembly: mscorlib.dll" in is malformed and not rendering properly.

#### framework/migration-guide/runtime-changes-in-the-net-framework-4-6-1.md
`VirtualizingStackPanel.IsVirtualizing` is gone.

### CODE REVIEW

File: docs/framework/migration-guide/net-compatibility-diagnostics.md
Comment (mairaw) [inline docs/framework/migration-guide/net-compatibility-diagnostics.md]: although the link now works, it seems to be an issue with the build system. why this method requires the *? I see that the page shows ms.assetid with the * but seems wrong somehow.

Comment (GuardRex) [inline docs/framework/migration-guide/net-compatibility-diagnostics.md]: Yep ... let me revert. The build report lied ... the original link works. https://docs.microsoft.com/en-us/dotnet/articles/framework/migration-guide/net-compatibility-diagnostics#42-enumerableemptytresult-always-returns-cached-instance

File: docs/framework/migration-guide/mitigation-cspparameters-parentwindowhandle-expects-an-hwnd.md
Comment (rpetrusha) [inline docs/framework/migration-guide/mitigation-cspparameters-parentwindowhandle-expects-an-hwnd.md]: This also should be `<xref:System.Security.AppContext>`.

File: docs/framework/migration-guide/runtime-changes-in-the-net-framework-4-6-2.md
Comment (rpetrusha) [inline docs/framework/migration-guide/runtime-changes-in-the-net-framework-4-6-2.md]: The xref to AppDomainAppPath should include `?displayProperty=fullname`

File: docs/framework/whats-new/obsolete-members.md
Comment (rpetrusha) [inline docs/framework/whats-new/obsolete-members.md]: @GuardRex, I had meant to address this issue for quite some time, but it's fallen on the back burner. I've finally started to investigate. AFAIK, the recommended alternative method doesn't actually exist in the release versions of the .NET Framework.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
@GuardRex, can you also add one more topic to this PR? In docs/framework/migration-guide/retargeting-changes-in-the-net-framework-4-7.md, there's a link to BringIndexInfoView(Int32) between the header and the table that should be removed.

File: docs/framework/whats-new/obsolete-members.md
Comment (GuardRex) [inline docs/framework/whats-new/obsolete-members.md]: It should be *GetDragDropMovedViewElements*: https://referencesource.microsoft.com/#System.Activities.Presentation/System.Activities.Presentation/System/Activities/Presentation/DragDropHelper.cs,673

... but *GetDragDropMovedViewElements* isn't public as the obsolete notes states ...
https://referencesource.microsoft.com/#System.Activities.Presentation/System.Activities.Presentation/System/Activities/Presentation/DragDropHelper.cs,710

**Comment by GuardRex** [comment]
@rpetrusha 

> In docs/framework/migration-guide/retargeting-changes-in-the-net-framework-4-7.md, there's a link to BringIndexInfoView(Int32) between the header and the table that should be removed.

I don't see it. Even a global search for "BringIndexInfoView" of all docs on my branch doesn't find it. Where is it?

**Comment by rpetrusha** [comment]
@GuardRex, I'm sorry -- it's in Runtime Changes in the .NET Framework 4.7 (docs/framework/migration-guide/runtime-changes-in-the-net-framework-4-7.md), and the API is BringIndexIntoView. I used to be able to type without error...

File: docs/framework/whats-new/obsolete-members.md
Comment (rpetrusha) [inline docs/framework/whats-new/obsolete-members.md]: @Guardrex, thanks for looking in the source.

Comment (GuardRex) [inline docs/framework/whats-new/obsolete-members.md]: @rpetrusha I performed a broader Internet search as well and didn't turn anything up on the depreciation of these. In a case like this, what's the next step? ... and do we hold this PR for further updates or does this become a new issue?

Comment (GuardRex) [inline docs/framework/whats-new/obsolete-members.md]: I guess the low-hanging fruit is to put a note in the project to address this. I'll do that now.

File: docs/framework/whats-new/obsolete-members.md
Comment (rpetrusha) [inline docs/framework/whats-new/obsolete-members.md]: The obsolete text should read, "This method does not support dragging multiple items."

File: docs/framework/whats-new/obsolete-members.md
Comment (rpetrusha) [inline docs/framework/whats-new/obsolete-members.md]: The obsolete text should red, "This method does not support dragging multiple items."

**Comment by GuardRex** [comment]
@rpetrusha I removed some stray spaces from lines 322 and 323. As you instructed, I only actually changed the *Message* column text for lines 319, 321, and 324.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
@GuardRex, this is ready to merge when the build completes.

**Comment by GuardRex** [comment]
@rpetrusha It choked on the change to ...
```
<xref:System.Security.AppContext>
```
Should we look to restore that documentation, or should we inline code the reference? I couldn't find it. I only see: https://referencesource.microsoft.com/#mscorlib/system/AppContext/AppContext.cs,24

The other one, `System.Windows.Controls.VirtualizingStackPanel.IsVirtualizing` was known and listed for further investigation in the project.

It's private ...

https://referencesource.microsoft.com/#PresentationFramework/Framework/System/Windows/Controls/VirtualizingStackPanel.cs,11126

Because the text is just mentioning virtualization in passing to make a behavioral point ...

> When an <xref:System.Windows.Controls.ItemsControl> displays a collection using virtualization (<xref:System.Windows.Controls.VirtualizingStackPanel.IsVirtualizing%2A> = `true`) and item-scrolling ...

... I suggest we inline code the ref with (set privately) on it ...

> When an <xref:System.Windows.Controls.ItemsControl> displays a collection using virtualization [`VirtualizingStackPanel.IsVirtualizing` = `true` (set privately)] and item-scrolling ...

... or using whatever alternative language you'd like to see there.

**Comment by GuardRex** [comment]
@rpetrusha I updated the ref to `System.Security.AppContext` to inline code, since that content doesn't exist in the API docs. I used ...

> [\`VirtualizingStackPanel.IsVirtualizing\` = \`true\` (set privately)]\`

... for the `IsVirtualizing` prop, which also doesn't exist due to its `private` access.

If those moves are ok, it should be ready to go.

**Comment by rpetrusha** [comment]
It's not System.Security.AppContext. It's System.AppContext. That should resolve in an <xref>. With that change, you can merge.

**Comment by GuardRex** [comment]
I had it that way initially. I must have misunderstood tho what you wanted changed: https://github.com/dotnet/docs/pull/1957#discussion_r117291487

**Comment by rpetrusha** [comment]
I think that I confused myself. I meant that it should also be System.AppContext, but that's not what I wrote.

**Pull request merged**
